scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -221,14 +229,26 @@ def net_start(args):
     netpath = args.netpath
     netname = xml_get_value(args.netpath, "name")
     print("defining network from {0}".format(netname))
-    networks = conn.listNetworks()
-    if netname not in networks:
-        print("defining network from {0}".format(netpath))
-        xml = readfile(netpath)
-        conn.networkDefineXML(xml)
-    print("starting {0} network".format(netname))
-    dom = conn.networkLookupByName(netname)
-    dom.create()
+
+    # Get the names of active and inactive network domains.
+    try:
```

Comment:
We could give a exit 1 while catching the exception gracefully :). Also logging would be great but we should think of logging later on. It would be too big a patch for the same.